metadata-db-app/src/cmr/metadata_db/api/provider.clj

```diff
@@ -2,8 +2,10 @@
   "Defines the HTTP URL routes for the application."
   (:require
    [cheshire.core :as json]
+   [clojure.string :as string]
```

Comment:
remove this require and the other one cmr.common.config, as they are not being used in the file